gallery/visualtests/testUtils.js

```diff
@@ -338,21 +342,26 @@ module.exports = {
           await focusElement.hover();
 
           if (waitForTooltip) {
-            await wait(500);
+            await wait(TOOLTIP_WAIT);
           }
 
           await checkScreenshotWithOutset(targetElement, outset);
         };
       },
 
-      focusAndHoverTest(elementSelector, hoverSelector = elementSelector, outset) {
+      focusAndHoverTest(elementSelector, hoverSelector = elementSelector, waitForTooltip = false, outset) {
```

Comment:
Note that I added `waitForTooltip` as the third param (rather than the fourth) to be consistent with `hoverTest`